tfrecorder/utils_test.py

```diff
@@ -75,7 +75,7 @@ def test_valid_records(self, mock_fn):
 
     mock_fn.return_value = {self.split: self.dataset}
     num_records = len(self.data['image'])
-
+    #omit dir='/tmp' if on windows
```

Comment:
You can remove `dir='/tmp'`. It should work on Linux as well.

Reply:
@cfezequiel Thanks for the comments,
I'll work on these changes and update the code, my apologies repeatedly miaaing out on things, hopefully after this first round of changes. I will understand the package better and things will become much smoother from here and on.